.changeset/tender-corners-stay.md

```diff
@@ -0,0 +1,7 @@
+---
+"fluid-framework": minor
```

Comment:
major, right?

Reply:
This is just a function of which branch the change is made on (so minor for everything currently), and not really used by our release process. Its intended to indicate what kind of version bump needs to be done when releasing, but we version bump after the previous release, not when processing these files so minor is correct (default for our main branch). Some details at https://github.com/microsoft/FluidFramework/wiki/Changesets#changesets

Reply:
I think that might be inaccurate and the changeset wiki doesn't really clarify what to do when there is only one branch (`main`). We have the choice of bumping major or minor just before a release. We should normally be bumping minor. Client group happens to be bumping major right now. 2.0.0-rc.3.0.0 to 2.0.0-rc.4.0.0 is a major prerelease bump. There is no rule that we couldn't bump 2.0.0-rc.4.0.0 to 2.0.0-rc.4.1.0 before 2.0.0-rc.4.0.0 is released.
If the changeset bump type indicates what bump must be in place for release. Since we always at least bump minor, a changeset with minor is always okay. If the changeset bump is major, then the release must be major. I would like to require that a PR with changeset of major is not mergeable until the version has had its major bump - that would prevent any -dev releases from being semantically invalid, but -dev is, well, special enough.
Since client group has already been bumped major it is okay to merge major bump changesets now.